### DISCUSSION TITLE
LF line endings on Windows?

### DISCUSSION THREAD

**Discussion by silviumc** [KDE/ghostwriter#776] [Q&A]
Hello,
Is there any setting to save files with LF line endings instead of CRLF on Windows?
Thanks

**Comment by silviumc** [comment]
An option to display line ending characters would be also nice. But I've seen that no feature requests are possible.

**Comment by HaaiHenkie** [comment]
Hello,

I also would like to have the possibility of UNIX style line endings under Windows. I now have to use Notepad++ to  convert the line endings after using Ghostwriter.

Cheers

**Comment by somethingSTRANGE** [comment]
I would also love to an option to force UNIX line endings when running on Windows. My organization stores markdown and most other text documents with UNIX line endings in version control and strict diffs (that take line endings into account) always show the entire file as being changed.

At the very least, it would be useful to have the existing line ending type detected when a document was opened and simply save using the same line ending type. As it is now, a document can start with LF endings, but it gets converted to CRLF as soon as any change is made to it in Ghostwriter.

**Comment by LostOnTheLine** [comment]
Because of this one missing necessary feature I find I can only use this App as a viewer. 
Which is a shame as it's quite good. 
I find myself opening in `Notepad++` to edit & `GhostWriter` to preview it after I change it. So I appreciate it automatically prompting a reload when it's saved. But I usually end up opening GhostWriter to the Right 3/4 of the screen, have Notepad++ on the left overlapping the editing side to make this workable, but sometimes the AutoSave gets turned back on & it seriously messes with the workflow. 
There are a few things that I'd like, like the ability to lock the sides so they scroll together or unlock them so they don't, or the ability to click on the preview & have it take you to the code line. But all of those are Nice-To-Haves, This feature IMO is the one thing making this generally unusable as a standalone editor working with anything Linux-Based, which in my experience is what most things that use Markdown are.

Back in the day, I'm not talking about it's origins with ASCII where they literally told a paper to return & feed a line, just it's results after it was a digital standard, Sun & Unix & Linux used just `LF` while Basic & Mac used `CR`, They were essentially saving a bit of space, said "why do we need 2?" & chose to keep a different half, Windows wanted to ensure compatibility so they used both so their files would be the same on any system, & essentially didn't want to mess with something that worked as it was for a meager benefit, & for text documents it wasn't an issue as the unused return would just be ignored. Now Mac is Unix-Based, so the only ones still using `CR` are Windows who, ironically, only did so for compatibility, which now is actually causing the incompatibility. Sure some things like E-Mail & HTML use `CRLF`, but they also all (to my knowledge) will automatically convert to it without issue, so it's not really in need of consideration. So if you need to support only 1 go with `LF`
